### ISSUE TITLE
Error when running rails console

### ISSUE THREAD

**Issue by aar0nr** [rails/web-console#184]
### Using:

* rails (4.2.5.1)
* web-console (3.1.0)

### Running:

```rails console```

### Outputs:

```
Running via Spring preloader in process 6529
~/.gem/ruby/2.3.0/gems/railties-4.2.5.1/lib/rails/railtie.rb:245:in `each_registered_block': undefined method `each' for nil:NilClass (NoMethodError)
  from ~/.gem/ruby/2.3.0/gems/railties-4.2.5.1/lib/rails/railtie.rb:224:in `run_console_blocks'
  from ~/.gem/ruby/2.3.0/gems/railties-4.2.5.1/lib/rails/application.rb:472:in `block in run_console_blocks'
  from ~/.gem/ruby/2.3.0/gems/railties-4.2.5.1/lib/rails/engine/railties.rb:13:in `each'
  from ~/.gem/ruby/2.3.0/gems/railties-4.2.5.1/lib/rails/engine/railties.rb:13:in `each'
  from ~/.gem/ruby/2.3.0/gems/railties-4.2.5.1/lib/rails/application.rb:472:in `run_console_blocks'
  from ~/.gem/ruby/2.3.0/gems/railties-4.2.5.1/lib/rails/engine.rb:438:in `load_console'
  from ~/.gem/ruby/2.3.0/gems/railties-4.2.5.1/lib/rails/commands/console.rb:58:in `initialize'
  from ~/.gem/ruby/2.3.0/gems/railties-4.2.5.1/lib/rails/commands/console.rb:9:in `new'
  from ~/.gem/ruby/2.3.0/gems/railties-4.2.5.1/lib/rails/commands/console.rb:9:in `start'
  from ~/.gem/ruby/2.3.0/gems/railties-4.2.5.1/lib/rails/commands/commands_tasks.rb:68:in `console'
  from ~/.gem/ruby/2.3.0/gems/railties-4.2.5.1/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
  from ~/.gem/ruby/2.3.0/gems/railties-4.2.5.1/lib/rails/commands.rb:17:in `<top (required)>'
  from ~/.gem/ruby/2.3.0/gems/activesupport-4.2.5.1/lib/active_support/dependencies.rb:274:in `require'
  from ~/.gem/ruby/2.3.0/gems/activesupport-4.2.5.1/lib/active_support/dependencies.rb:274:in `block in require'
  from ~/.gem/ruby/2.3.0/gems/activesupport-4.2.5.1/lib/active_support/dependencies.rb:240:in `load_dependency'
  from ~/.gem/ruby/2.3.0/gems/activesupport-4.2.5.1/lib/active_support/dependencies.rb:274:in `require'
  from ~/.gem/ruby/2.3.0/gems/activesupport-4.2.5.1/lib/active_support/dependencies.rb:268:in `load'
  from ~/.gem/ruby/2.3.0/gems/activesupport-4.2.5.1/lib/active_support/dependencies.rb:268:in `block in load'
  from ~/.gem/ruby/2.3.0/gems/activesupport-4.2.5.1/lib/active_support/dependencies.rb:240:in `load_dependency'
  from ~/.gem/ruby/2.3.0/gems/activesupport-4.2.5.1/lib/active_support/dependencies.rb:268:in `load'
  from ~/.rubies/ruby-2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
  from ~/.rubies/ruby-2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
  from -e:1:in `<main>'
```

**Comment by gsamokovarov** [comment]
Can you give us a bit more context? Are you sure `web-console` is the cause of that error?

**Comment by aar0nr** [comment]
Yes, I created a brand new rails app, and when I update web-console to ```v3.1.0``` I am unable to start a rails console session via ```bin/rails console```. If I downgrade to ```v3.0.0``` it works fine.

**Comment by joelmichael** [comment]
I am also sure it is the cause of the error.

```
[243, 252] in /Users/joel/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/railties-4.2.5.1/lib/rails/railtie.rb
   243:       klass = self.class
   244:       while klass.respond_to?(type)
   245:         if klass == Object && type.to_s == 'console'
   246:           debugger
   247:         end
=> 248:         klass.public_send(type).each(&block)
   249:         klass = klass.superclass
   250:       end
   251:     end
   252:   end
(byebug) Object.method :console
#<Method: Class(Kernel)#console>
(byebug) Object.method(:console).source_location
["/Users/joel/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/web-console-3.1.0/lib/web_console/extensions.rb", 9]
(byebug) 
```

**Comment by kapso** [comment]
+1 getting the same issue

**Comment by prathamesh-sonpatki** [comment]
@gsamokovarov A similar issue was reported on https://github.com/rails/rails/issues/23300. I created sample app - https://github.com/prathamesh-sonpatki/web-console-console-issue, please check.

**Comment by Eric-Guo** [comment]
+1, I can confirm, it's due to web-console instead of Rails master branch, if using web-console v3.0.0, such error not happen, no matter using puma 2.16.0 or rails master, so must a bug between v3.0.0 and v3.1.0

**Comment by Eric-Guo** [comment]
Confirm revert #182 will resolve this issue.

**Comment by gsamokovarov** [comment]
Holy moly. Thank you folks for the investigation. I have a fix and will release 3.1.1 ASAP.

**Comment by aar0nr** [comment]
Thank you @gsamokovarov , I appreciate the quick fix!
